.travis.yml

```diff
@@ -58,8 +58,10 @@ script:
         else
             for example in examples/*/ ; do
                 examplename=$(basename "$example")
-                mkdir -p "${HOME}/.foldercache/${exercise}/${examplename}/.stack-work"
-                ln -f -s "${HOME}/.foldercache/${exercise}/${examplename}/.stack-work"
+                exercisename=$(basename "$exercise")
```

Comment:
You are right...but we'll have to rewrite this soon to fix #397, so I'll just leave it like this for now.